common/achievements/gotta-screep.ts

```diff
@@ -0,0 +1,39 @@
+import {RowComponent} from '../components'
+import query from '../components/query'
+import SleepingEffect from '../status-effects/sleeping'
+import {achievement} from './defaults'
+import {Achievement} from './types'
+
+const GottaSchreep: Achievement = {
+	...achievement,
+	numericId: 54,
+	id: 'gotta_screep',
+	levels: [
+		{
+			name: 'Gotta Screep',
+			description: 'KO a Hermit while your active Hermit is sleeping.',
+			steps: 1,
+		},
+	],
+	onGameStart(game, player, component, observer) {
+		let isSleeping = false
+
+		observer.subscribe(player.hooks.onTurnStart, () => {
```

Comment:
I think this should just be checked on KO

Reply:
That does not work. If you KO on the same turn bdubs falls asleep, it actually counts as bdouble0 being asleep because the sleep status effect is applied before the KO is checked. The second test tests this case.